requirements.txt

```diff
@@ -6,3 +6,5 @@ configparser==3.5.0
 pdfkit==0.6.1
 lxml==4.1.1
 requests==2.18.4
+ipython>=5.0.0,<6
```

Comment:
Is this required?

Reply:
ipython is used by androguard, it should be installed by androguard , but I got a case where it wasn't so I add it by security

Reply:
Can you please confirm if it's not installed by androguard and is a requirement for androguard strings extraction function to work?

Reply:
Normally we could remove it from requierements.txt   will redo all the tests to confirm , it is correctly installed by androguard

Reply:
we can remove ipython>=5.0.0,<6 in the requirement list ak-androguard do install it correctly 

will update PR

Reply:
will update the file , ak-androguard do install it correctly no need to add it, iw will introduce a risk if we forget to update it if needed